scene/main/viewport.cpp

```diff
@@ -170,6 +170,11 @@ Ref<Image> ViewportTexture::get_image() const {
 }
 
 void ViewportTexture::_setup_local_to_scene(const Node *p_loc_scene) {
+	_setup_local_to_scene_imp(p_loc_scene);
+	vp_pending = false;
```

Comment:
I would add a comment as otherwise it's not clear why this needs to be here instead of at the end of the `setup_local_to_scene_imp` mthod.
```suggestion
	// Always reset this, even if previous call fails with an error.
	vp_pending = false;
```